grpc_collector_client.go

```diff
@@ -91,7 +91,7 @@ func (client *grpcCollectorClient) ConnectClient() (Connection, error) {
 
 		grpcClient, ok := uncheckedClient.(cpb.CollectorServiceClient)
 		if !ok {
-			return nil, fmt.Errorf("Grpc connector factory did not provide valid client!")
+			return nil, fmt.Errorf("grpc connector factory did not provide valid client")
```

Comment:
nit: I have noticed that every error message inside of the tracer uses capital an exclamation. I definitely prefer the way you changed it, but it feels like it would be better to have a separate PR that changes all the errors at the same time.